src/Sylius/Bundle/ThemeBundle/Tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Sylius\Bundle\ThemeBundle\Tests\DependencyInjection;
+
+use Matthias\SymfonyConfigTest\PhpUnit\ConfigurationTestCaseTrait;
+use Sylius\Bundle\ThemeBundle\DependencyInjection\Configuration;
+
+/**
+ * @author Kamil Kokot <kamil.kokot@lakion.com>
+ */
+class ConfigurationTest extends \PHPUnit_Framework_TestCase
+{
+    use ConfigurationTestCaseTrait;
+
+    /**
+     * @test
+     */
+    public function it_uses_app_themes_as_default_themes_location()
+    {
+        $this->assertProcessedConfigurationEquals(
+            [
+                [],
+            ],
+            [
+                'locations' => ['%kernel.root_dir%/themes'],
+            ]
+        );
+    }
+
+    /**
+     * @test
+     */
+    public function it_does_not_add_default_theme_location_if_there_are_some_defined_by_user()
+    {
+        $this->assertProcessedConfigurationEquals(
+            [
+                [
+                    'locations' => ['/custom/path', '/custom/path2'],
+                ],
+            ],
+            [
+                'locations' => ['/custom/path', '/custom/path2'],
+            ]
+        );
+    }
+
+    /**
+     * @test
+     */
+    public function it_uses_the_last_theme_locations_passed_and_rejects_the_other_ones()
+    {
+        $this->assertProcessedConfigurationEquals(
```

Comment:
:+1:, I worked on processing theme configuration and these arrays took enormously much space.